src/pages/api/chat.ts

```diff
@@ -0,0 +1,56 @@
+import { LangChainStream, Message, streamToResponse } from 'ai';
+import { AIMessage, HumanMessage } from 'langchain/schema'
+import { NextApiRequest, NextApiResponse } from 'next';
+import { ChatOpenAI } from '@langchain/openai';
+
+export default async function handler(req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  const { messages } = await req.body;
+  const currentMessageContent = messages[messages.length - 1].content;
+
+  const response = await fetch("https://deploy-preview-12424--ethereumorg.netlify.app/api/vector", {
```

Comment:
Hardcoded for now

To investigate:
https://docs.netlify.com/configure-builds/environment-variables/
https://docs.netlify.com/configure-builds/environment-variables/#deploy-urls-and-metadata